src/utils/setup.middleware.js

```diff
@@ -55,7 +55,9 @@ exports.initLocals = function( req, res, next ) {
 			{ title: 'How you can help', href: '/page/how-you-can-help' },
 			{ title: 'How businesses and organizations can help', href: '/page/how-businesses-and-organizations-can-help' },
 			{ title: 'Experienced families', href: '/page/experienced-families' },
-			{ title: 'Weekend Family Connections', href: '/page/weekend-family-connections' }
+			{ title: 'Weekend Family Connections', href: '/page/weekend-family-connections' },
+			{ title: 'The Neighborhood', href: '/page/the-neighborhood' },
```

Comment:
The link works as intended (I believe), but when I navigate to that page there's no content:
<img width="1247" alt="image" src="https://user-images.githubusercontent.com/5726612/153673988-14fb9356-e113-440c-8ae1-4821552f7ce5.png">